core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -195,9 +195,10 @@ contract Liquidatable is PricelessPositionManager {
                 sponsor: sponsor,
                 liquidator: msg.sender,
                 state: Status.PreDispute,
-                rawLockedCollateral: positionToLiquidate.rawCollateral,
+                lockedCollateral: _getCollateral(positionToLiquidate.rawCollateral),
                 tokensOutstanding: positionToLiquidate.tokensOutstanding,
                 liquidatedCollateral: positionCollateral.sub(positionToLiquidate.withdrawalRequestAmount),
+                rawFeeAttenuation: _convertCollateral(FixedPoint.fromUnscaledUint(1)),
```

Comment:
If we pass in a hard coded "1" here, then should we be naming the parameter to _convertCollateral "collateral"? I think this is the only place _convertCollateral is called

Reply:
What should we be calling it instead? This function does the opposite of the _getCollateral function, so I feel like not calling it collateral would feel even stranger, but definitely open to suggestions.

Reply:
Do you anticipate using `_convertCollateral` in other situations? Why not just define `_inverseFeeMultiplier` to make it more clear what you are doing?

Reply:
I used it in other locations as you suggested.